reporter/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -65,13 +65,13 @@ class CopyrightStatementsProcessor {
     companion object {
         private val YEAR_PLACEHOLDER = "<ORT_YEAR_PLACEHOLDER_TRO>"
         private val KNOWN_PREFIX_REGEX = listOf(
-                "^([C|c]opyrighted \\(c\\))",
-                "^([C|c]opyrighted)",
-                "^([C|c]opyright \\(c\\))",
+                "^(\\(c\\))",
```

Comment:
Commit message nits:
- "as *the* result"
- "Remove *this* hidden assumption *by* always *getting*"
- "as *the* result" (again)
- Last sentence: "As ordering can be arbitrary now, use a more readable one."